.pre-commit-config.yaml

```diff
@@ -15,29 +15,35 @@
 # limitations under the License.
 #
 repos:
--   repo: https://github.com/ambv/black
-    rev: 19.10b0
-    hooks:
-    -   id: black
-        language_version: python3
-
--   repo: https://github.com/asottile/seed-isort-config
+  - repo: https://github.com/asottile/seed-isort-config
     rev: v1.9.3
     hooks:
-    -   id: seed-isort-config
-
--   repo: https://github.com/pre-commit/mirrors-isort
+      - id: seed-isort-config
+  - repo: https://github.com/pre-commit/mirrors-isort
     rev: v4.3.21
     hooks:
-    -   id: isort
-
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.2.3
+      - id: isort
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.770
+    hooks:
+      - id: mypy
+  - repo: https://github.com/peterdemin/pip-compile-multi
+    rev: v1.5.8
     hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
-    -   id: check-docstring-first
-    -   id: check-added-large-files
-    -   id: check-yaml
+      - id: pip-compile-multi-verify
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-docstring-first
+      - id: check-yaml
         exclude: ^helm/superset/templates/
-    -   id: debug-statements
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+        args: ["--markdown-linebreak-ext=md"]
```

Comment:
I've been tearing my hair out with the `trailing-whitespace` hook. The hook works locally but complains in a remote setting stating that `README.md` needs reformatting albeit the fact that by construction the environments/package versions are equivalent. The only way I was able to resolve this was with the  `--markdown-linebreak-ext` argument.